dates.py

```diff
@@ -6,10 +6,10 @@
 import sys
 
 # SEM_BEGIN=datetime.datetime.now(pytz.timezone('Asia/Kolkata'))
-SEM_BEGIN = build_event.generateIndiaTime(2020, 1, 2, 0, 0)
-MID_TERM_BEGIN = build_event.generateIndiaTime(2020, 2, 17, 0, 0)
-MID_TERM_END = build_event.generateIndiaTime(2020, 2, 25, 23, 59)
-END_TERM_BEGIN = build_event.generateIndiaTime(2020, 4, 20, 0, 0)
+SEM_BEGIN = build_event.generateIndiaTime(2020, 9, 1, 0, 0)
+MID_TERM_BEGIN = build_event.generateIndiaTime(2020, 9, 19, 0, 0)
```

Comment:
```suggestion
MID_TERM_BEGIN = build_event.generateIndiaTime(2020, 10, 23, 0, 0)
```

Reply:
What's wrong with the MID_TERM_BEGIN Date?